jdaviz/configs/mosviz/plugins/parsers.py

```diff
@@ -283,11 +286,10 @@ def _parse_as_spectrum1d(path):
                 except IORegistryError:
                     data = _parse_as_spectrum1d(data)
 
-            # Copy (if present) region to top-level meta object
-            if ('header' in data.meta and
-                    'S_REGION' in data.meta['header'] and
-                    'S_REGION' not in data.meta):
-                data.meta['S_REGION'] = data.meta['header']['S_REGION']
+            # Make metadata layout conform with other viz.
+            if 'header' in data.meta:
```

Comment:
With this change, we should no longer worry about `meta[somekey]` vs `meta['header'][somekey]`.

It seems like the `meta['header']` stuff was imposed by `specutils`, so in the future if we add new parsing logic that loads stuff from `specutils`, we will need to remember to do this along with the new code. A little messy but seems to work. 🤷